Makefile

```diff
@@ -5,7 +5,7 @@ build:
 	cabal build
 
 test:
-	cabal test unit-tests
+	cabal test unit-tests --test-show-details=streaming --test-option=--format=checks --test-option=--times --test-option=--color
```

Comment:
As mentioned in [slack](https://teamfossa.slack.com/archives/G01DUC2JW3B/p1646683694688619) thread